Dockerfile

```diff
@@ -1,28 +1,35 @@
 FROM debian:buster
 
-WORKDIR /dhx
+ARG CHAIN_VERSION
+ENV _CHAIN_VERSION=${CHAIN_VERSION}
+RUN echo "DataHighway chain version ${_CHAIN_VERSION}"
 
-RUN apt-get update && apt-get install -y build-essential wget cmake pkg-config libssl-dev openssl git clang libclang-dev curl vim unzip screen bash \
-	&& wget -O - https://sh.rustup.rs | sh -s -- -y --default-toolchain nightly-2020-02-17 \
-	&& curl https://sh.rustup.rs -sSf | sh -s -- -y \
-	&& PATH=$PATH:/root/.cargo/bin \
-	&& rustup update stable && rustup toolchain install nightly-2020-02-17 && rustup target add wasm32-unknown-unknown --toolchain nightly-2020-02-17 \
-	&& cargo version 
+WORKDIR /dhx
 
+# FIXME - only copy necessary files to reduce size of image, and try using intermediate stages again
 COPY . .
 
-RUN PATH=$PATH:/root/.cargo/bin \
-	&& curl https://getsubstrate.io -sSf | bash -s -- --fast \
-	&& ./scripts/init.sh \
-  && cargo build --release \
+RUN apt-get update && apt-get install -y build-essential wget cmake pkg-config libssl-dev \
+    openssl git gcc clang libclang-dev curl vim unzip screen bash \
+    && curl https://getsubstrate.io -sSf | bash -s -- --fast \
+    && . ~/.cargo/env \
+    && wget -O - https://sh.rustup.rs | sh -s -- -y \
+    && PATH=$PATH:/root/.cargo/bin \
+    && rustup update stable \
+    && rustup update nightly \
+    && rustup toolchain install nightly-2020-02-17 \
```

Comment:
the nightly toolchain needs to stay on this version from back in February still?

Reply:
yes, until we update the version of Substrate that we're using just to play it safe